src/backend/marsha/core/management/commands/dev_simulate_reminders.py

```diff
@@ -0,0 +1,156 @@
+"""Send reminders mails with simulated data to test the actual rendering."""
+from datetime import timedelta
+import os
+import uuid
+
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from marsha.core.defaults import IDLE, RAW, RUNNING
+from marsha.core.factories import LiveRegistrationFactory, VideoFactory
+
+
+class Command(BaseCommand):
+    """Send reminders for scheduled webinar."""
+
+    help = "Send samples of reminder mails for scheduled webinar."
+
+    def handle(self, *args, **options):
+        """Execute management command."""
+
+        if os.environ.get("DJANGO_CONFIGURATION") == "Development":
```

Comment:
Using `settings.DEBUG` is better I think. And if it's not in debug mode you exit the command

```
if not settings.DEBUG:
   raise CommandError

# Execute the command
```